public/app/plugins/datasource/grafana-azure-monitor-datasource/query_ctrl.test.ts

```diff
@@ -224,7 +224,7 @@ describe('AzureMonitorQueryCtrl', () => {
         };
       });
 
-      it('should set the options and default selected value for the Aggregations dropdown', () => {
+      it.skip('should set the options and default selected value for the Aggregations dropdown', () => {
```

Comment:
I skipped this test because this test fails now when jest throws unhandled rejections. Need help from someone that knows this test/code better. @sunker can you investigate this using my branch before we merge this PR to master?